design-documents/graph-ql/coverage/eav/EavGraphQl.graphqls

```diff
@@ -0,0 +1,42 @@
+# Copyright © Magento, Inc. All rights reserved.
+# See COPYING.txt for license details.
+
+type Query {
+    customAttributeMetadata(attributes: [AttributeInput!]!): CustomAttributeMetadata @resolver(class: "Magento\\EavGraphQl\\Model\\Resolver\\CustomAttributeMetadata") @doc(description: "The customAttributeMetadata query returns the attribute type, given an attribute code and entity type") @cache(cacheable: false)
```

Comment:
this is good but please change the location of your file to this file
design-documents/graph-ql/coverage/custom-attributes/attributes-metadata.graphqls

https://github.com/magento/architecture/pull/456/files#diff-113bf18a368d3d5c2ee6e9956737276d3f7fcc4306fe477c8dc67f937d0b8839R1